WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -491,6 +555,24 @@ private extension ProductsViewController {
 
         toolbar.isHidden = filters.numberOfActiveFilters == 0 ? isEmpty : false
     }
+
+    func showBottomToolbar() {
+        tabBarController?.tabBar.isHidden = true
+
+        // trigger safe area update
+        if let tabBarController {
+            let currentFrame = tabBarController.view.frame
+            tabBarController.view.frame = currentFrame.insetBy(dx: 0, dy: 1)
+            tabBarController.view.frame = currentFrame
+        }
```

Comment:
Nice trick, maybe we can wrap this in a small extension helper!

Reply:
I'll keep it inline for now and hope safe areas will "just work" going forward 🙂
But feel free to convert it to extension if you spot similar issue in other views.